app/views/unattended/provisioning_templates/iPXE/preseed_default_ipxe.erb

```diff
@@ -8,7 +8,7 @@ oses:
 - Ubuntu
 %>
 <% if @host.operatingsystem.name == 'Debian' -%>
-  <%- keyboard_params = "auto=true domain=#{@host.domain}" -%>
+  <%- keyboard_params = "auto=true "-%>
```

Comment:
```suggestion
  <%- keyboard_params = "auto=true" -%>
```

Reply:
yes, i was aware of that but somehow removed it during squashing to put everything into a single commit - sorry for the additional unnecessary push

now I only have to run the tests like you mentioned